mappings/net/minecraft/world/gen/feature/StructurePoolFeatureConfig.mapping

```diff
@@ -1,6 +1,7 @@
 CLASS net/minecraft/class_3812 net/minecraft/world/gen/feature/StructurePoolFeatureConfig
-	COMMENT A feature config that specifies a starting pool and a size for the first two parameters of
-	COMMENT {@link net.minecraft.structure.pool.StructurePoolBasedGenerator#addPieces(net.minecraft.util.Identifier, int, net.minecraft.structure.pool.StructurePoolBasedGenerator.PieceFactory, net.minecraft.world.gen.chunk.ChunkGenerator, net.minecraft.structure.StructureManager, net.minecraft.util.math.BlockPos, java.util.List, java.util.Random, boolean, boolean)}.
+	COMMENT A feature config that specifies a starting pool and a size for {@linkplain
+	COMMENT net.minecraft.structure.pool.StructurePoolBasedGenerator#method_30419
```

Comment:
I can't forget how many javadoc errors have arisen from this single link due to their refactors, but is this one right for 1.16.5?

Reply:
This one is indeed right.